src/components/devices/kostal_sem/counter.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+	<div class="device-kostal_sem-counter">
```

Comment:
Eigentlich wird die Klasse nicht verwendet und ist nur der Vollständigkeit halber eingetragen. Sie sollte jedoch einheitlich im Kebab-Stil benannt werden.
```suggestion
	<div class="device-kostal-sem-counter">
```